pkg/cloud/gcp/gcp.go

```diff
@@ -262,6 +262,10 @@ func (c *GcpCloud) SetupInstances(zone, networkName, sshPublicKey, ami string, s
 				Scheduling: &compute.Scheduling{
 					AutomaticRestart: &automaticRestart,
 				},
+				Labels: map[string]string{
```

Comment:
can we do the same with static ip names?

Reply:
sure